doc/crm.8.adoc

```diff
@@ -1380,6 +1380,92 @@ Usage:
 status [ring|quorum|qdevice|qnetd|cpg]
 .........
 
+[[cmdhelp_corosync_link,Knet link management]]
+==== `link`
+
+Knet is a multi-link and multi-protocol transport used by corosync.
+This level provides subcommands for managing these links.
+
+All the subcommands in this level operates on local configuration file.
```

Comment:
Upon successfully changing the configuration file locally, it will be synced to all cluster nodes.